### DISCUSSION TITLE
Error al cargar la pagina

### DISCUSSION THREAD

**Discussion by ahranza** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#285] [Q&A]
Hola! Ya subí el repositorio pero no carga la página. 
Me sale esto:
![2022-03-06 (5)](https://user-images.githubusercontent.com/99292356/156971966-025e15ff-9822-4a4c-b71d-8069b216b433.png)

Ya revisé y sí están los archivos. Cómo lo soluciono?

**Comment by OscarPedroza** [comment]
Hola, esta cargando desde la raiz del repositorio, para cargar la pagina html tienes que navegar a la carpeta donde se encuentra desde las URL, por ejemplo, por ejemplo, si se encuentra en una carpeta llamada proyecto, tienes que acceder a esa o varias carpetas más y después al archivo de html.

**Comment by MarcosSilvaG** [comment]
Hola! ya creaste tu repositorio y subiste todos los archivos? (pregunto porque en tu perfil no aparecen)

**Reply by ahranza** [reply]
Lo borré y lo subí de nuevo y no funcionó.

**Comment by GloriaCoralCerecedo** [comment]
Mira yo lo que hice es que la página de inicio le puse index.html. las imágenes en una carpeta, el css si son varios en una carpeta y si tienes más HTML igual en otra carpeta.

**Reply by ahranza** [reply]
Ya la cambié pero no funcionó

**Comment by JuanSalvador73** [comment]
¿Subiste todos los archivos directamente, o subiste un folder con esos mismos archivos?  Es probable que te falte alguno.

**Comment by ahranza** [comment]
Borré el repositorio y lo volví a subir con sus indicaciones. Me sigue saliendo lo mismo. Será error de los archivos?

**Reply by hackit92** [reply]
me compartes la liga de tu repositorio? Asi te puedo ayudar más rápido

**Reply by OscarPedroza** [reply]
¿Si nombraste tu archivo con un .github.io al final para cargarlo con Git Hub Pages?

**Reply by ahranza** [reply]
https://github.com/ahranza/htmlpasteleria.git

**Reply by hackit92** [reply]
@ahranza si veo tus archivos y en general está bien, el problema es que utilizaste rutas absolutas para llamar tus archivos (C:/Users/Aranza/etc) mientras que debería ser algo como (./assets/) eso puede ser un problema en un futuro, si puedes corregirlo en tus htmls sería bueno, en cuanto al despliegue en github creo que tiene que ver con que está en una carpeta dentro de la raíz

**Reply by ahranza** [reply]
Lo de la raíz es cambiar a main?

**Reply by hackit92** [reply]
> 

Asi es, solo saca todo de la carpeta html, varios publicaron un pequeño tutorial para publicar en github pages pero te dejo un enlace aquí 
https://www.youtube.com/watch?v=sLTNgxxSBR4 
Aún después de hacer estas modificaciones no se desplegarán imagenes y css debido a lo de las rutas que te comento en tu código, tocará arreglarlas para que se despliegue bien

**Reply by JorgeLReyes** [reply]
> Lo de la raíz es cambiar a main?

Puedes poner la ruta de la siguiente manera por ejemplo para imagenes "./imagenes/nombreimagen.png"

En si el punto indica la raiz y de ahi a que carpeta tiene que acceder para encontrar el archivo

**Reply by ahranza** [reply]
Eso lo pongo en el código en visual verdad?

**Reply by JorgeLReyes** [reply]
si por ejemplo en tu logo seria < img src=".\imagenes\L'Après-Midi.png" alt="logo pasteleria" width="400px">

**Reply by JorgeLReyes** [reply]
nadamás quita el espacio entre < img (lo puse porque de otra manera git piensa que quiero poner una imagen)

**Reply by ahranza** [reply]
Muchas gracias! Creo que ya se ve

**Comment by CesarMezaB** [comment]
Para las rutas de las imágenes debe ser la siguiente  en el documento de Index debe ser "carpeta/Nombreimagen.jpg" ejemplo "/img/pastel.jpg"

**Reply by ahranza** [reply]
Cambio la ruta en visual?

**Reply by CesarMezaB** [reply]
Si desde el editor del código y luego subes el los archivos de nuevo

**Comment by CesarMezaB** [comment]
Ejemplo de tus imágenes `<img src= /imágenes/nombreImagen alt="pastel" >` también faltó cerrar la etiqueta de imagen y poner las comillas

**Reply by ahranza** [reply]
Muchas gracias! Ya las cambié

**Comment by JLuisPrz** [comment]
Hola, recibe tu repo y solo es buscar bien en donde se encuentra el index.html
Te dejo aquí el link de tu [pagina](https://ahranza.github.io/htmlpasteleria/html/Index.html) .
Tus imágenes no las carga, creo que solo es cuestión que revises bien las rutas

**Reply by ahranza** [reply]
Hola, ese link lo puedo sacar de la parte pages? O es otro? Porque ya hice un repositorio nuevo y esperé y sigue saliendo igual, aún con todos los cambios que me dijeron.

**Reply by JLuisPrz** [reply]
Yo lo saque viendo como esta estructurado tu repositorio, aquí esta el nuevo link a tu [página](https://ahranza.github.io/htmlpasteleria.github.io/html/Index.html)  
Ahora si se ven la imagenes

**Reply by ahranza** [reply]
:O Muchas gracias T_T

**Reply by JLuisPrz** [reply]
De nada :D

**Comment by manusanabria** [comment]
En tu repositorio selecciona **Settings**, sección **Pages**, en **Source** cambia a la rama main, **Save** y se generará en enlace a tu página.

**Reply by ahranza** [reply]
Me sale este: https://ahranza.github.io/htmlpasteleria.github.io/
Pero sigo viéndolo mal.

**Reply by manusanabria** [reply]
El problema está en que en los archivos se encuentran en una carpeta html, necesitas moverlos a la raíz del repositorio

**Reply by ahranza** [reply]
Muchas gracias! Creo que ya se ve

**Comment by carlos-gmx3** [comment]
Oh ya, creo que lo subiste como un repositorio normal y no has configurado el despliegue de dicho repositorio
Tienes que checarlo en settings del mismo repo
https://www.codecademy.com/article/f1-u3-github-pages
![image](https://user-images.githubusercontent.com/99064239/157171232-636ff80b-dcc5-4465-8894-e31a19305daa.png)
Simplemente le indicas que despliegue la rama principal y mueves todo al directorio raiz

**Reply by ahranza** [reply]
Sí lo cambié y luego hice un repositorio nuevo y se veía igual.
![m ](https://user-images.githubusercontent.com/99292356/157173101-b412c43c-40f4-441e-bd32-c6d19c3c57fc.png)

**Reply by hackit92** [reply]
@ahranza no esta disponible el enlace, podrías decir cual es el problema actual?

**Reply by ahranza** [reply]
El link que me sale https://ahranza.github.io/htmlpasteleria.github.io/

Me da error al abrirlo. Pero según yo sí está bien el repositorio.

**Reply by carlos-gmx3** [reply]
Tal vez en despliegue es case-sensitive
Renombra tu Index.html a index.html
Todo en minúsculas pues
Espera 10min y trata de entrar

**Reply by ahranza** [reply]
Ya lo cambié. Ahora no veo las imágenes nada mas. Creo que es la forma en que busco el link.

**Reply by carlos-gmx3** [reply]
De hecho, el ./ Está de más
Si el archivo html está en la misma carpeta usa el nombre de archivo y su extensión NombreArchivo.jpg. Si está en otra carpeta, NombreCarpeta/NombreArchivo.jpg
Creo q el ./ es para regresar un nivel de carpeta pero por la organización de tu repo no funciona

**Comment by ChanLuis** [comment]
Creo que debes cambiar los links "./ejemplo.html" por los links de github por archivo

**Reply by ahranza** [reply]
El que me da la pag https://ahranza.github.io/htmlpasteleria.github.io/ no es el que tendría que mandar? O lo saco de otro lado?
